src/styles/card.scss

```diff
@@ -0,0 +1,100 @@
+$main-color: #060b35;
+$secondary-color: #616070;
+$button-color: #00acdc;
+$white: #fff;
+
+.card {
+  font-family: Roboto, sans-serif;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background-color: $white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  &:nth-child(-n + 4) {
+    margin-bottom: 46px;
+  }
+  &__image {
+    width: 160px;
+    height: 134px;
+    align-self: center;
+    margin-bottom: 40px;
+  }
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-color;
+    margin-bottom: 4px;
+  }
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary-color;
+    font-weight: 400;
+  }
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-top: 16px;
+    &-counter {
+      font-size: 10px;
+      line-height: 14px;
+      font-weight: 400;
+      color: $main-color;
+    }
+  }
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 24px;
+    &-label {
+      font-size: 12px;
+      line-height: 14px;
+      font-weight: 500;
+      color: $secondary-color;
+    }
+    &-price {
+      font-size: 16px;
+      line-height: 18px;
+      font-weight: 700;
+      color: $main-color;
+    }
+  }
+  &__purcache-button {
+    display: flex;
+    margin-top: 16px;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    color: $white;
+    text-transform: uppercase;
+    text-decoration: none;
+    background-color: $button-color;
+    text-align: center;
+    padding: 11px 70px 11px 69px;
+    border: 1px solid transparent;
+    border-radius: 5px;
+    &:hover {
+      background-color: $white;
+      color: $button-color;
+      border-color: $button-color;
+    }
+  }
+}
+
+.stars {
+  display: flex;
+  &__star {
+    background: url(../images/star.svg) no-repeat 50% 50%;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    &:nth-child(-n + 4) {
+      background-image: url(../images/star-active.svg);
+    }
```

Comment:
the key idea is to be the initial functionality of `stars` BEM-block (i.e. with all those modifiers), and then apply for-loop, see the checklist, p.12